src/main/java/org/isf/utils/jobjects/DateTextField.java

```diff
@@ -175,9 +176,12 @@ public GregorianCalendar getCompleteDate(){
 			year.setText("");
 			return null;
 		}
-		date.set(GregorianCalendar.DAY_OF_MONTH,getDay());
-		date.set(GregorianCalendar.MONTH,getMonth()-1);
-		date.set(GregorianCalendar.YEAR,getYear());
+		date.set(Calendar.DAY_OF_MONTH,getDay());
+		date.set(Calendar.MONTH,getMonth()-1);
+		date.set(Calendar.YEAR,getYear());
+		date.set(Calendar.HOUR_OF_DAY, 23);
+		date.set(Calendar.MINUTE, 59);
+		date.set(Calendar.SECOND, 59);
```

Comment:
As said in comments on OP-482, this is accettable as temporary solution, could we add a comment for this? Thanks!

Reply:
@mwithi FWIW,  I added the comment.